rust/Cargo.toml.in

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 rust-version = "1.63.0"
 
 [workspace]
-members = [".", "./derive"]
+members = [".", "./derive", "./libhtp-rs"]
```

Comment:
`libhtp-rs` seems a bit redundant now. Could it just be `libhtp`, or even `htp` to match the package name?

Reply:
ok for htp